plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt

```diff
@@ -134,6 +134,11 @@ However, if you still insist on skipping versions, we strongly advise to read al
 
 == Upgrading from Platform 4.4.0 to ${org.exoplatform.platform.version} ==
 
+=== Description of the plugins used in this upgrade ===
+
+# ECMS-7595: Cannot download a document : the Download button is not visible
```

Comment:
I think the text line should be: *PLF-7186 Search Indexing Management page does not appear after PLF upgrade*, this not relate to issue ECMS-7595